docs/azure/integrations-overview.md

```diff
@@ -104,6 +75,183 @@ The same is true for SQL and PostgreSQL services:
 
 For more information on the difference between run and publish modes, see [.NET Aspire app host: Execution context](xref:dotnet/aspire/app-host#execution-context).
 
+### APIs for expressing Azure resources in different modes
+
+The distributed application builder, part of the [app host](../fundamentals/app-host-overview.md), employs the builder pattern, allowing consumers to `AddAzure*` resources to the [_app model_](../fundamentals/app-host-overview.md#terminology). In addition to simply adding resources, developers can also express configuration and how said resource behave in various execution contexts. Azure hosting integrations provide APIs to specify how these resources should be "published" and "run".
+
+When the app host executes, the [_execution context_](../fundamentals/app-host-overview.md#execution-context) determines whether the app host is in <xref:Aspire.Hosting.DistributedApplicationOperation.Run> or <xref:Aspire.Hosting.DistributedApplicationOperation.Publish> mode. The naming conventions for these APIs indicate the intended action for the resource. For more information on execution modes, see [Execution context](../fundamentals/app-host-overview.md#execution-context).
+
+The following table summarizes the naming conventions used to express Azure resources:
+
+| Operation | API | Description |
+|--|--|--|
+| Publish | `PublishAsConnectionString` | Changes the resource to be published as a connection string reference in the manifest. |
+| Publish | `PublishAsExisting` | Uses an existing Azure resource when the application is deployed instead of creating a new one. |
+| Run | `RunAsContainer` | Configures an equivalent container to run locally. For more information, see [Local containers](#local-containers). |
+| Run | `RunAsEmulator` | Configures the Azure resource to be emulated. For more information, see [Local emulators](#local-emulators). |
+| Run | `RunAsExisting` | Marks the resource as an existing resource when the application is running. |
+| Publish and Run | `AsExisting` | Marks the resource as an existing resource regardless of the operation. |
```

Comment:
```suggestion
| Run | `RunAsExisting` | Uses an existing resource when the application is running instead of creating a new one. |
| Publish and Run | `AsExisting` | Uses an existing resource regardless of the operation. |
```